site/content/en/docs/contrib/tests.en.md

```diff
@@ -294,7 +294,7 @@ Steps:
 asserts basic "service" command functionality
 
 Steps:
-- Create a new `k8s.gcr.io/echoserver` deployment
+- Create a new `registry.k8s.io/echoserver` deployment
```

Comment:
The echoserver no longer lives in the main registry, it has been forked to multiple locations.

Reply:
For minikube, I think `k8s.gcr.io/echoserver:1.4` was replaced with `kicbase/echo-server:1.0`